proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties

```
@@ -15,39 +15,38 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-velocity.error.already-connected=Jesteś już połączony z tym serwerem!
-velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connected=Już jesteś połączony z tym serwerem!
+velocity.error.already-connected-proxy=Już jesteś połączony z tym proxy!
 velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
 velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
-velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
+velocity.error.connecting-server-error=Nie udało Ciebie połączyć z {0}. Spróbuj ponownie później.
```

You deleted too many `się` :P
```suggestion
velocity.error.connecting-server-error=Nie udało się Ciebie połączyć z {0}. Spróbuj ponownie później.
```